src/common/meta/src/util.rs

```diff
@@ -27,10 +27,27 @@ pub fn get_prefix_end_key(key: &[u8]) -> Vec<u8> {
     vec![0]
 }
 
+/// Get next prefix key of `key`.
+#[inline]
+pub fn get_next_prefix_key(key: &[u8]) -> Vec<u8> {
+    let mut key = Vec::from(key);
```

Comment:
Maybe more efficient:
```suggestion
    let mut key = Vec::with_capacity(key.len() + 1);
    key.extend_from_slice(key);
```